nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -73,6 +73,10 @@ void RegulatedPurePursuitController::configure(
 
   global_path_pub_ = node->create_publisher<nav_msgs::msg::Path>("received_global_plan", 1);
   carrot_pub_ = node->create_publisher<geometry_msgs::msg::PointStamped>("lookahead_point", 1);
+  projected_pose_pub_ =
```

Comment:
The projected pose looks mostly for debugging value. Is there a reason to keep this in the final form? I can understand publishing the cross track error

Reply:
Yes this is for debugging only, so I could add a parameter to enable and disable it. It helps to visualize where the cross-track error is being calculated from.

- [ ] Make projected pose pub configurable

Reply:
I meant more a development artifact, but if you think its useful as a life debugging instrument you think folks will use, no problem. Might want to rename the topic to give some more context then